payjoin-relay/src/main.rs

```diff
@@ -0,0 +1,176 @@
+use std::sync::Arc;
+
+use anyhow::{Context, Result};
+use axum::body::Bytes;
+use axum::extract::Path;
+use axum::http::{Request, StatusCode};
+use axum::routing::{get, post};
+use axum::Router;
+use payjoin::v2::MAX_BUFFER_SIZE;
+use tracing_subscriber::filter::LevelFilter;
+use tracing_subscriber::EnvFilter;
+
+mod db;
+use crate::db::DbPool;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    init_logging();
+    let pool = DbPool::new(std::time::Duration::from_secs(30)).await?;
+    let ohttp = Arc::new(init_ohttp()?);
+    let ohttp_config = ohttp_config(&*ohttp)?;
+    let target_resource = Router::new()
+        .route(
+            "/:id",
+            post({
+                let pool = pool.clone();
+                move |id, body| post_fallback(id, body, pool)
+            }),
+        )
+        .route(
+            "/:id/receive",
+            get({
+                let pool = pool.clone();
+                move |id| get_request(id, pool)
+            })
+            .post({
```

Comment:
this is basically me as a receiver say "I accept payjoin" ?

Reply:
This is how a receiver returns a payjoin proposal